cost-analyzer/values.yaml

```diff
@@ -423,6 +423,44 @@ oidc:
     #     claimValues:
     #       - "editor"
 
+teams:
+  # teamsConfigMapName: kubecost-rbac-teams-config
+  # teamsConfig:
```

Comment:
thoughts on adding usage comments on these fields so it documents this a bit?

Reply:
haha I just noticed that right before, already pushed